### DISCUSSION TITLE
KEEPER Spring Rest Docs 라이브러리 제작기(1)

### DISCUSSION THREAD

**Discussion by 02ggang9** [BDD-CLUB/challenge-every-week-posting#369] [1. 개발 관련 글 모음]
안녕하세요.. 이번주는 두레 블로그 글을 써서 개인 블로그 글 작성이 조금 힘들었네요..!

아래는 초안이고 다음주까지 완성해서 수정하도록 하겠습니다.  

[KEEPER - KEEPER Spring Rest Docs DSL 제작기 (1)](https://02ggang9.github.io/keeper/keeperSPringRestDocs1/)

아직까지는 KEEPER에 핏한 코드이고, 시간과 기회가 된다면 라이브러리로 만들고 싶네요..!!

**Comment by gusah009** [comment]
좋네요~ 👍👍
